pkg/infra/game/stage/updateItems/updateItems.go

```diff
@@ -0,0 +1,20 @@
+package updateItems
```

Comment:
Maybe this should be in the same package as loot (albeit this doesn't exist yet)?